pages/articles/docker.md

```diff
@@ -0,0 +1,201 @@
+# Using PnP PowerShell in Docker containers
+
+Using Docker allows you to use any software inside virtual envirionments, without having to install this software directly on your laptop or server. These virtual environments are containers, which might be imagined as lightweight virtual machines. Having their own virtual disks, memory and processes, containers are rationally isolated from your laptop operating system, providing you a great way of experimenting and using any version of PnP.PowerShell without installing it.
+
+![Using software in virtual environments, called containers](./../images/docker/dockercontainers.png)
+
+The good news is that you will not even need to install PnP.PowerShell in containers by your own: PnP team is already publishing Docker container images for each supported and nightly release, [here](https://hub.docker.com/r/m365pnp/powershell). You will however need to install docker runtime.
+
+If you use Windows, we would recommend you using Linux containers with help of WSL. Alternatively, even though it is not the most common way, you might want to run PnP.PowerShell in Windows containers. Mind you that some use cases might be limited when using Windows containers. If you use Mac OS or Linux, the easiest way is to use Linux containers.
+
+## Play with PnP.PowerShell online
+
+You can try using m365pnp/powershell Docker containers online, without installing anything on your desktop/laptop.
+
+1. Open https://labs.play-with-docker.com/
+
+2. Login
+
+3. Click `Start`
+
+4. Click `+ ADD NEW INSTANCE`
+
+5. Run in the online console:
+
+    ```
+    docker run --rm -it m365pnp/powershell
+    ```
+
+After that you can start running commands like `Connect-PnPOnline`.
+
+## Installing Docker locally
+
+\<to be verified\>
```

Comment:
Can we please remove this section, the `to be verified` one ? If we are not sure, lets not add it and confuse users